purchase_line_procurement_group/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 {
     "name": "Purchase Line Procurement Group",
     "summary": "Group purchase order line according to procurement group",
-    "version": "11.0.1.0.0",
+    "version": "11.0.1.1.0",
```

Comment:
I see only "fix" commits: shall we go for 1.0.1?

Reply:
Actually the "generation of stock move" part is more an improvement, as it wasn't considered before. Moreover, it's already used in one of our projects with 11.0.1.1.0 so let's keep it like that.